src/schemas/protocol.schema.v1.json

```diff
@@ -200,6 +201,12 @@
             "$ref": "#/definitions/Item"
           }
         },
+        "introductionPanel" : {
```

Comment:
`items` is for array types; I think this should read

```
"introductionPanel": {
  "$ref": "#/definitions/IntroductionPanel"
},
```

Reply:
I updated this, hopefully I did it right. :)

Reply:
It probably doesn't matter, but I don't think `"type": "object"` is needed here (adjacent to the #ref), since the `IntroductionPanel` type reference defines the object type. (example: "skipLogic", below.)